thoth/storages/graph/postgres.py

```diff
@@ -290,12 +290,13 @@ def run_software_environment_listing(
         self, start_offset: int = 0, count: int = 100, is_user_run: bool = False
     ) -> list:
         """Get listing of software environments available for run."""
-        return self._do_software_environment_listing(start_offset, count, is_user_run, "RUNTIME")
+        software_environment = SoftwareEnvironment()
+        return self._do_software_environment_listing(start_offset, count, is_user_run, environment_type=software_environment.RUNTIME)
```

Comment:
Does it need to be instantiated?

What about:
```python
SoftwareEnvironment.RUNTIME
```

as done bellow?

Reply:
Actually it's a WIP because it's not working: 
```
Query ---> run_software_environment_listing()
Traceback (most recent call last):
  File "query_test.py", line 386, in <module>
    result = graph.run_software_environment_listing()
  File "/home/fmurdaca/work/aicoe/storages/thoth/storages/graph/postgres.py", line 294, in run_software_environment_listing
    return self._do_software_environment_listing(start_offset, count, is_user_run, environment_type=software_environment.RUNTIME)
AttributeError: 'SoftwareEnvironment' object has no attribute 'RUNTIME'
```